src/main/java/stsjorbsmod/powers/PathosPower.java

```diff
@@ -0,0 +1,73 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDiscardAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+
+import java.util.Iterator;
+
+public class PathosPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(PathosPower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    public PathosPower(final AbstractCreature owner, final int numberOfTurns) {
+        super(STATIC);
+
+        this.owner = owner;
+        this.amount = numberOfTurns;
+        this.isTurnBased = true;
+
+        updateDescription();
+    }
+
+    @Override
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        this.flash();
+        AbstractCard c = AbstractDungeon.returnRandomCurse();
+        addToBot(new MakeTempCardInDiscardAction(c, 1));
+    }
+
+    @Override
+    public void onInitialApplication() {
+        Iterator var1 = AbstractDungeon.player.hand.group.iterator();
+
+        while(var1.hasNext()) {
+            AbstractCard c = (AbstractCard)var1.next();
+            c.setCostForTurn(-9);
+        }
+    }
+
+    @Override
+    public void onCardDraw(AbstractCard card) {
```

Comment:
I investigated how to deal with the temp cards being added to your hand in the middle of a turn; I think the most straightforward option would be to replace this onCardDraw hook usage with a new `SpirePostfixPatch` on `CardGroup::addToHand` that does the same `card.setCostForTurn(-9)`; it looked like that was going to cover all of the different temp-card-adding actions I could see.

I considered whether using `freeToPlayOnce` would be better than using `setCostForTurn(-9)`; I think what you're already doing is correct/better, since we want to avoid making X cost cards free to play (for consistency with corruption).

Reply:
That was quite doable, thanks for the suggestion. I'm real inexperienced with patching mechanisms, so let me know if I did anything poorly.